### PR TITLE
added landing page, not done

### PR DESCRIPTION
**Pull request by almir0201**

### CODE REVIEW

**Comment by Shekey** [comment]
Comments:
1. You have overflow so you want to prevent scroll, 1 solution can be to add overflow-hidden to whole section (DESKTOP)
2. "Find all your favourite pokemon" space between lines are not matched, and font feels awkward, the real question if font even works
3. Min height of page should be 100vh, I see some blank space at the bottom (DESKTOP)
4. Make with ❤️ for the PokéSpartans team Platzi Master and whole footer needs to follow header width size, it needs to match exaclty
5. Instead of using "class="absolute w-[18%] lg:w-full max-w-[133px] lg:max-w-[167px] ml-[7%] top-[4%] lg:mt-3 lg:ml-9"

you could replace margins with left-0 and top-9, it makes more sense



Overall: pretty close, fix those things above and you can merge it
@almir0201

**Comment by almir0201** [comment]
Close #121

**Comment by almir0201** [comment]
@Shekey  for easier review here are issues 
image

File: src/components/ui/Hero/Hero.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    window.addEventListener('resize', handleResizeWindow);
+    return () => {
+      window.removeEventListener('resize', handleResizeWindow);
+    };
+  }, []);
+  return (
+    <div className={className}>
+      <VerticalLayout align="start">
+        <Headline
+          type={`${width > 1023 ? 'h1' : 'h5'}`}
```
Comment (Shekey) [inline src/components/ui/Hero/Hero.js:21]: why not just width>1023 ? xx : xx

File: src/components/ui/Hero/Hero.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          className="text-black font-karla tracking-widest !leading-7xl">
+          {headline}
+        </Headline>
+        <Paragraph
+          variant={`${width > 1279 ? '2xl' : 'xl'}`}
```
Comment (Shekey) [inline src/components/ui/Hero/Hero.js:26]: same as above

File: src/components/ui/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { cn } from '../../../lib/utils';
+import { VerticalLayout } from '../VerticalLayout/VerticalLayout';
+import { Paragraph } from '../Paragraph/Paragraph';
+
+export const Footer = ({ labelRight, labelLeft, className }) => {
+  return (
+    <div className="container mt-auto mb-8">
+      <div className={cn(className)}>
```
Comment (Shekey) [inline src/components/ui/Footer/Footer.js:8]: just className if you don't have nothing to combine with

**Review by Shekey — CHANGES_REQUESTED** [review]

**Review by Shekey — APPROVED** [review]

**Pull request merged**